kubernetes/elastic-helm/daemonset.yaml

```
@@ -12,6 +12,7 @@ presets:
 image:
   repository: docker.elastic.co/beats/elastic-agent
   tag: 8.16.0-SNAPSHOT
+  pullPolicy: always
```

```suggestion
  pullPolicy: Always
```
Reference: https://kubernetes.io/docs/concepts/containers/images/#image-pull-policy